src/pluto/helpers.py

```diff
@@ -1,51 +1,56 @@
-import boto3
 import re
 import pandas as pd
 from datetime import datetime
 import json
 from typing import Dict
-import os
 from dotenv import load_dotenv
-from zipfile import ZipFile
-from io import BytesIO
+from src.digital_ocean_client import DigitalOceanClient
 
 load_dotenv()
 
 BUCKET_NAME = "edm-publishing"
+REPO_NAME = "db-pluto"
 
 
 def get_data(branch) -> Dict[str, pd.DataFrame]:
     rv = {}
-    url = f"https://edm-publishing.nyc3.digitaloceanspaces.com/db-pluto/{branch}/latest/output/qaqc"
-    rv["df_mismatch"] = csv_from_DO(f"{url}/qaqc_mismatch.csv")
-    rv["df_null"] = csv_from_DO(f"{url}/qaqc_null.csv")
-    rv["df_aggregate"] = csv_from_DO(f"{url}/qaqc_aggregate.csv")
-    rv["df_expected"] = csv_from_DO(
-        f"{url}/qaqc_expected.csv", kwargs={"converters": {"expected": json.loads}}
+    url = f"https://edm-publishing.nyc3.digitaloceanspaces.com/db-pluto/{branch}/latest/output"
+
+    client = digital_ocean_client()
+
+    kwargs = {"true_values": ["t"], "false_values": ["f"]}
+    rv["df_mismatch"] = client.csv_from_DO(
+        url=f"{url}/qaqc/qaqc_mismatch.csv", kwargs=kwargs
+    )
+    rv["df_null"] = client.csv_from_DO(url=f"{url}/qaqc/qaqc_null.csv", kwargs=kwargs)
+    rv["df_aggregate"] = client.csv_from_DO(
+        url=f"{url}/qaqc/qaqc_aggregate.csv", kwargs=kwargs
+    )
+    rv["df_expected"] = client.csv_from_DO(
+        url=f"{url}/qaqc/qaqc_expected.csv",
+        kwargs={"converters": {"expected": json.loads}} | kwargs,
     )
-    rv["df_outlier"] = csv_from_DO(
-        f"{url}/qaqc_outlier.csv", kwargs={"converters": {"outlier": json.loads}}
+    rv["df_outlier"] = client.csv_from_DO(
+        url=f"{url}/qaqc/qaqc_outlier.csv",
+        kwargs={"converters": {"outlier": json.loads}} | kwargs,
     )
 
-    pluto_corrections_zip = zip_from_DO(
+    pluto_corrections_zip = client.zip_from_DO(
         zip_filename=f"db-pluto/{branch}/latest/output/pluto_corrections.zip",
-        bucket=BUCKET_NAME,
     )
 
-    rv["pluto_corrections"] = unzip_csv(
+    rv["pluto_corrections"] = client.unzip_csv(
         csv_filename="pluto_corrections.csv", zipfile=pluto_corrections_zip
     )
 
-    rv["pluto_corrections_applied"] = unzip_csv(
+    rv["pluto_corrections_applied"] = client.unzip_csv(
         csv_filename="pluto_corrections_applied.csv", zipfile=pluto_corrections_zip
     )
-    rv["pluto_corrections_not_applied"] = unzip_csv(
+    rv["pluto_corrections_not_applied"] = client.unzip_csv(
         csv_filename="pluto_corrections_not_applied.csv", zipfile=pluto_corrections_zip
     )
```

Comment:
Might be nice to decompose this to it's own function? Hard to glance and see that some lines set a variable and others use that variable to populate the dictionary

Reply:
I'm a little confused by what you mean here - which thing do you think should be its own function - just the stuff that comes from the zipfile or something else?

Reply:
Sorry, let me elaborate. I was thinking of something like

```python
corrections_from_DO(rv, client)
```
and then 
```python
def corrections_from_DO(rv, client):
    pluto_corrections_zip = client.zip_from_DO(
        zip_filename=f"db-pluto/{branch}/latest/output/pluto_corrections.zip",
    )
    rv["pluto_corrections"] = client.unzip_csv(
        csv_filename="pluto_corrections.csv", zipfile=pluto_corrections_zip
    )
etc...
```
My concern is that at a glance there are one client methods that is used to set a variable and another client method that uses that a variable to set a value in `rv`. They are all "parallel" here (idk if that's the best term) and it might be more readable if this `zip_from_DO` -> variable -> `unzip_csv` -> `rv` process was bundled together. Alternatively the client could handle all this? Maybe I'm way overthinking this, I think the existing process is definitely good enough to merge

Reply:
i'm always pro making functions smaller, happy to do that